plop-templates/component.tsx.hbs

```diff
@@ -2,9 +2,19 @@ import React from "react";
 import Styled{{name}} from "./{{name}}.styles";
 
 export interface {{name}}Props {
+  /**
+  * `children` can be any `ReactNode`
```

Comment:
Thought; my general opinion around comments is that they should be value-add. For example; they should add extra context that the name and type doesn't provide. If it's simply repeating the name and type (childen / react node) then it's not worth imho.